amp-beta-tester.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Plugin Name: AMP Beta Tester
+ * Description: Opt-in to receive non-stable release builds for the AMP plugin.
+ * Plugin URI: https://amp-wp.org
+ * Author: AMP Project Contributors
+ * Author URI: https://github.com/ampproject/amp-wp/graphs/contributors
+ * Version: 0.1
+ * Text Domain: amp-beta-tester
+ * Domain Path: /languages/
+ * License: GPLv2 or later
+ *
+ * @package AMP Beta Tester
+ */
+
+namespace AMP_Beta_Tester;
+
+define( 'AMP__BETA_TESTER__DIR__', dirname( __FILE__ ) );
+
+add_action( 'plugins_loaded', __NAMESPACE__ . '\init' );
+
+/**
+ * Hook into WP.
+ *
+ * @return void
+ */
+function init() {
+	// Abort init if AMP plugin is not available.
+	if ( ! defined( 'AMP__FILE__' ) ) {
+		add_action( 'admin_notices', __NAMESPACE__ . '\show_missing_amp_notice' );
+		return;
+	}
+}
+
+/**
+ * Display an admin notice if the AMP plugin is not detected.
+ *
+ * @return void
+ */
+function show_missing_amp_notice() {
+	$error = sprintf(
+		// translators: %s: AMP plugin URL.
+		esc_html__(
+			'AMP Beta Tester requires AMP to be installed and active. You can download AMP %s.',
+			'amp-beta-tester'
+		),
+		'<a href="https://amp-wp.org" target="_blank">here</a>'
```

Comment:
Since the AMP Beta Tester plugin is distributed _with_ the AMP plugin pre-release builds, there would not be a possibility for the AMP plugin to not be installed. So I suggest this message can be simplified to just advising that the AMP plugin needs to be active, and the link can be removed.

Reply:
Worth noting that any notice in this file will not be translatable by users as the file is not part of the stable build and thus not available on the WordPress.org translation platform.

To solve this:

1. Add any translatable strings needed in this file also to `amp.php`.
2. Use the `amp` text domain in all cases

For example, we'd add this to the bottom of `amp.php`:

```php
__( 'AMP Beta Tester requires AMP to be installed and active. You can download AMP %s.', 'amp );
```

Then it will be translatable by users and part of the language packs downloaded by WordPress.

In `amp-beta-tester.php` you could then use the same string as usual.